### ISSUE TITLE
Fastly::FullAuthRequired exception generated when trying to create Fastly Client with API key.

### ISSUE THREAD

**Issue by Caslyn** [fastly/fastly-ruby#60]
The README file indicates there's an option to create a Fastly client with an API key, then right after makes a call to fastly.current_user, which in this case will raise an exception.

ReadMe.md

```
# use {api_key: 'your-key'} or {user: 'username', password: 'password'} as login options
fastly = Fastly.new(login_opts)

current_user     = fastly.current_user
```

fastly.rb

```
  # Return a User object representing the current logged in user.
  # NOTE: requires you to be fully authed - will not work with only an API key
  def current_user
    fail FullAuthRequired unless fully_authed?
    @current_user ||= get(User)
  end
```

To help users avoid this exception, can we clarify the README file or omit the calls to current_user/user right after we give them the option to log in/create Fastly client with API key?

**Comment by mmay** [comment]
Understood. Feel free to open a PR with README improvements. Otherwise, we'll try to make some improvements the next time we release a new version.

**Comment by Integralist** [comment]
Closing this issue now we've published v4.0.0 https://rubygems.org/gems/fastly
